ec2/ec2.go

```diff
@@ -787,15 +777,35 @@ type IPPerm struct {
 	Protocol     string              `xml:"ipProtocol"`
 	FromPort     int                 `xml:"fromPort"`
 	ToPort       int                 `xml:"toPort"`
-	SourceIPs    []string            `xml:"ipRanges>item>cidrIp"`
```

Comment:
I couldn't find a test change that exercises the reason why this is being changed (what you described in the pull request summary). Perhaps it's here and I'm just not seeing it?